glean-core/src/ping/mod.rs

```diff
@@ -241,6 +241,26 @@ impl PingMaker {
             );
         }
 
+        // Due to the way the experimentation identifier could link datasets that are intentionally unlinked,
```

Comment:
🤔  yes, I suppose it would because the `metrics` section of the ping would not be empty. I should probably ignore adding the id if there is nothing else in the `metrics` or `events` unless `send_if_empty` is set.